### PR TITLE
Integrated has paid function

### PR DESCRIPTION
**Pull request by spiftire**
Basic logic. not tested. NEEDS TESTING

### CODE REVIEW

**Comment by spiftire** [comment]
closes #93

File: server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java [hunk 1]
```diff
@@ -75,4 +76,30 @@ public boolean isAdminOfOrganization(Long organizationId) {
...
+        try {
+            member = query.getSingleResult();
+        } catch (NoResultException e) {
+            e.getMessage();
+            return Response.status(Response.Status.NO_CONTENT).entity("Is not a member").build();
+        }
+
+        if ( !member.hasPaid()) {
+            return Response.status(Response.Status.NO_CONTENT).entity("Has NOT paid").build();
+        }
```
Comment (0us) [inline server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java]: "Is not a member" og "Has NOT paid" bør bruke forskjellige Status-koder

File: server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java [hunk 2]
```diff
@@ -75,4 +76,30 @@ public boolean isAdminOfOrganization(Long organizationId) {
 
         return result;
     }
+
+    @PermitAll
+    public Response harMemberPaid(Long organizationId, User user) {
```
Comment (markusranda) [inline server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java]: "har"memberPaid

Should we really mix Norwegian and English in our naming scheme?

**Comment by spiftire** [comment]
Bitte liten typo 😅

fre. 22. nov. 2019, 19:52 skrev Markus Randa <notifications@github.com>:

File: server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java (hunk 2 above)
Comment (spiftire) [reply server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java]: Burde vel heller ikke være permitAll

Comment (spiftire) [reply server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java]: done

File: server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java (hunk 1 above)
Comment (spiftire) [reply server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java]: done

**Pull request merged**